### PR TITLE
Add aria-label to My Account Navigation Item

### LINKED ISSUE #43625: Accessibility: My Account nav unlabeled
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

The navigation menu on the My Account page is unlabeled. See:
https://github.com/woocommerce/woocommerce/blob/758df4854d7b627fc26eb5c0860dc492dea6a06e/plugins/woocommerce/templates/myaccount/navigation.php#L25

### Expected behavior

There should be an aria-label on the nav tag. Something like:
`<nav class="woocommerce-MyAccount-navigation" aria-label="Account Pages">`

### Actual behavior

No label is present.

### Steps to reproduce

See code linked above.

### WordPress Environment

N/A

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

### PR DESCRIPTION
**Pull request by masteradhoc**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:
Add the appropriate aria-label to the My Account Navigation Item

Closes #43625 .

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to the My Account Page, Login
2. Check if the Aria-Label is now present

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [x] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [x] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 
Add aria-label to account page

#### Comment 

</details>

### CODE REVIEW

**Comment by masteradhoc** [comment]
@nielslange would it be possible to get this reviewed? :)

File: plugins/woocommerce/templates/myaccount/navigation.php [hunk 1]
```diff
@@ -22,7 +22,7 @@
 do_action( 'woocommerce_before_account_navigation' );
 ?>
 
-<nav class="woocommerce-MyAccount-navigation">
+<nav class="woocommerce-MyAccount-navigation" aria-label="Account Pages">
```
Comment (nielslange) [inline plugins/woocommerce/templates/myaccount/navigation.php]:
```suggestion
<nav class="woocommerce-MyAccount-navigation" aria-label="Account pages">
```

Nitpick: The majority of `aria-label` in our codebase are using sentence case. What are your thoughts on using this here as well?

**Review by nielslange — CHANGES_REQUESTED** [review]
Thanks for your contribution, @masteradhoc, and apologies for my delayed response. 

The change looks good. I just left a minor suggestion regarding using _"Account pages"_ instead of _"Account Pages"_.

While reviewing the PR, I noticed a few failing CI checks, which, I believe, you cannot see. 

1. **Highlight template changes**: As your change affects a template, the version number of the template should be bumped. This shows merchants/developers that the template changed. Currently, the version is set to `2.6.0`. It should be bumped to `9.0.0`, expecting that this change will be shipped with that release.
2. **Changelog entry**: It seems there was an issue with the changelog entry creation:
    - You've selected `Patch` and `Tweak - A minor adjustment to the codebase.` However, the `Patch` checkbox appears broken.
    - The option to `Automatically create a changelog entry from the details below` is unchecked. This checkbox triggers the workflow to automatically create the changelog entry.
    - Please add a simple changelog entry, such as "Add aria-label to account pages".

Could you please make these updates? Feel free to reach out if you have any questions or need further assistance!

**Comment by masteradhoc** [comment]
> Thanks for your contribution, @masteradhoc, and apologies for my delayed response.
> 
> The change looks good. I just left a minor suggestion regarding using _"Account pages"_ instead of _"Account Pages"_.
> 
> While reviewing the PR, I noticed a few failing CI checks, which, I believe, you cannot see.
> 
> 1. **Highlight template changes**: As your change affects a template, the version number of the template should be bumped. This shows merchants/developers that the template changed. Currently, the version is set to `2.6.0`. It should be bumped to `9.0.0`, expecting that this change will be shipped with that release.
> 2. **Changelog entry**: It seems there was an issue with the changelog entry creation:
>    
>    * You've selected `Patch` and `Tweak - A minor adjustment to the codebase.` However, the `Patch` checkbox appears broken.
>    * The option to `Automatically create a changelog entry from the details below` is unchecked. This checkbox triggers the workflow to automatically create the changelog entry.
>    * Please add a simple changelog entry, such as "Add aria-label to account pages".
> 
> Could you please make these updates? Feel free to reach out if you have any questions or need further assistance!

@nielslange thanks for the help on this one! i went ahead and accepted your typo fix, adjusted the template version and fixed my changelog issues in the PR description. :)

**Comment by nielslange** [comment]
Thank you so much for the quick changes, @masteradhoc. Due to an issue with the CI pipeline, I had to close and re-open the PR. So, please don't worry when you're seeing this. I'll wait until the CI pipeline is done and will do another review then.

File: plugins/woocommerce/templates/myaccount/navigation.php [hunk 2]
```diff
@@ -22,7 +22,7 @@
 do_action( 'woocommerce_before_account_navigation' );
 ?>
 
-<nav class="woocommerce-MyAccount-navigation">
+<nav class="woocommerce-MyAccount-navigation" aria-label="Account pages">
```
Comment (barryhughes) [inline plugins/woocommerce/templates/myaccount/navigation.php]: Curious, should this be translated (as with other user-facing text)?

Comment (masteradhoc) [reply plugins/woocommerce/templates/myaccount/navigation.php]: @barryhughes made the requested change - makes definately sense.

**Review by barryhughes — APPROVED** [review]
Great, thank you for making that adjustment :+1:

(No other notes from my end.)

**Review by nielslange — APPROVED** [review]

**Comment by adimoldovan** [comment]
Bypassing the failing php checks to merge this one. More info p1715596214628469/1715181240.581999-slack-C03CPM3UXDJ.

**Comment by nielslange** [comment]
@masteradhoc Thanks again for your contribution! 🙌 And thanks to @adimoldovan, who just merged it on my behalf.

**Pull request merged**